CMakeLists.txt

```diff
@@ -97,7 +97,7 @@ if (DOWNLOAD_GMP)
             INTERFACE_INCLUDE_DIRECTORIES "${INSTALL_DIR}/include")
     add_dependencies(GMP::GMP gmp)
 else()
-    find_package(GMP)
+    find_package(GMP REQUIRED)
```

Comment:
Required is missing here on purpose to provide a nicer error message, please drop this change.